design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -144,6 +144,287 @@ Note:
   </ros2_control>
 ```
 
+```xml
+  <ros2_control name="12DOF_Legged_System_Robot_MultiInterface" type="system">
+    <hardware>
+      <classType>ros2_control_demo_hardware/2DOF_System_Hardware_MultiInterface_MultiWrite</classType>
```

Comment:
here also capitalized CamelCase:

"2DoFSystemHardwareMultiInterfaceMultiWrite"

Reply:
Oh, here we should use different name:

either: "ros2_control_demo_hardware/12DoFSystemHardwareMultiInterfaceMultiWrite"

or event better:

"talos_control/TalosSystemHardware" or "talos_control/TalosRobotHardware" (choose one you prefer)

Of course we should than create `talos_control` package at some point.

Reply:
Will do